Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -187,5 +186,117 @@ def proximal(self, x, tau, out = None):
 
         return soft_shrinkage(x, tau, out = out) 
 
+class WeightedL1Norm(Function):
+    r"""WeightedL1Norm function
+            
+            .. math:: F(x) = ||x||_{\ell^1(w)} (w is array of positive weights)
+    
+    Parameters:
+    -----------
+
+    weight: weight array matching the size of the wavelet coefficients, default None
+        If None returns the regular L1Norm.
+    b: data, default None
+        Translation of the function.
+    """
+
+    def __new__(cls, weight=None, b=None):
+            
+        if weight is None:
+            return L1Norm(b=b)
+        else:
+            return super(WeightedL1Norm, cls).__new__(_WeightedL1Norm)
+
+class _WeightedL1Norm(WeightedL1Norm): 
+           
+    def __init__(self, weight=None, b=None):
```

Comment:
should the argument be `weights`?

Reply:
In `WeightedL2NormSquared` it is `weight`. 

https://github.com/TomographicImaging/CIL/blob/ff4741092b7f10bde5e0572c0ac103288a163ffc/Wrappers/Python/cil/optimisation/functions/L2NormSquared.py#L154